recipes/keybert/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "keybert" %}
+{% set version = "0.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/keybert-{{ version }}.tar.gz
+  sha256: cfa7d7d106cbf21268e7f62107e639a5d831fd4e146237b8453e90a211b72c70
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
```

This can be noarch. A failing CI on Windows because of a missing dependency is ok.
```suggestion
  noarch: python 
```